### ISSUE TITLE
Following quickstart gave me error

### ISSUE THREAD

**Issue by wotaen** [jorgebastida/gordon#3]
Hi there, 
I'm following quickstart (http://gordon.readthedocs.io/en/latest/quickstart_py.html) and at the end when I do `gordon apply`, I get this:

```
Applying project...
  0001_p.json (cloudformation)
    ✓ No updates are to be performed.
  0002_pr_r.json (custom)
Traceback (most recent call last):
  File "/usr/local/bin/gordon", line 9, in <module>
    load_entry_point('gordon==0.2.0', 'console_scripts', 'gordon')()
  File "/usr/local/lib/python3.4/dist-packages/gordon/bin.py", line 97, in main
    getattr(obj, options.func)()
  File "/usr/local/lib/python3.4/dist-packages/gordon/core.py", line 432, in apply
    getattr(self, 'apply_{}_template'.format(template_type))(name, filename, context)
  File "/usr/local/lib/python3.4/dist-packages/gordon/core.py", line 476, in apply_custom_template
    outputs = template.apply(context, self)
  File "/usr/local/lib/python3.4/dist-packages/gordon/actions.py", line 131, in apply
    action_outputs[action.name] = action.apply(context, project)
  File "/usr/local/lib/python3.4/dist-packages/gordon/actions.py", line 198, in apply
    file_hash = utils.get_file_hash(self.filename)
  File "/usr/local/lib/python3.4/dist-packages/gordon/utils.py", line 77, in get_file_hash
    return get_zip_hash(filename)
  File "/usr/local/lib/python3.4/dist-packages/gordon/utils.py", line 70, in get_zip_hash
    digest.update(path)
TypeError: Unicode-objects must be encoded before hashing
```

Thanks

**Comment by jorgebastida** [comment]
This is a bug related to python3. Will try to fix it asap

**Comment by chrislopresto** [comment]
I got the same error with Python 3.5.1. Made it through successfully with Python 2.7.11. Thanks for the tip @jorgebastida

**Comment by jorgebastida** [comment]
I've managed to fix all integration tests in python3. These changes are now and master and will be released as part of ``0.2.1``.
